ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -257,6 +263,42 @@ export default class TestDataModal extends React.Component {
     this.closeModal();
   };
 
+  selectableTestClassName = test => {
+    if (this.hasDifferentUnit(test)) {
+      return 'bg-warning';
+    }
+    return '';
+  };
+
+  selectableTestTitle = test => {
```

Comment:
I think this:
```js
  selectableTestTitle = test => {
    return this.hasDifferentUnit(test)
      ? `Warning: ${this.getOriginalTestName(
          test,
        )} has a different measurement unit (${test.measurementUnit})`
      : this.getOriginalTestName(test);
  };
```
would be more readable.

Reply:
I'm not sure we'll stick to 2 output types in the future. We may need to stack multiple types of details into this title, which can require more statements/expressions than the ternary operator.

Reply:
OK, it's just a matter of form. Do how you think it's better.